x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning.tsx

```diff
@@ -34,40 +40,195 @@ interface Props {
+        >
+          {i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.invalidBoostsErrorMessage',
+            {
+              defaultMessage:
+                'One or more of your boosts is no longer valid, possibly due to a schema type change. Delete any old or invalid boosts to dismiss this alert.',
+            }
+          )}
+        </EuiCallOut>
+      )}
+      {unsearchedUnconfirmedFields && (
+        <EuiCallOut
+          color="warning"
+          iconType="alert"
+          title={i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.uncofirmedFieldsBannerLabel',
+            {
+              defaultMessage: 'Recently added fields are not being searched by default',
+            }
+          )}
+          data-test-subj="RelevanceTuningUnsearchedFieldsCallout"
+        >
+          <FormattedMessage
+            id="xpack.enterpriseSearch.appSearch.engine.relevanceTuning.uncofirmedFieldsErrorMessage"
+            defaultMessage="If these new fields should be searchable, turn them on here by toggling Text Search. Otherwise, confirm your new {schemaLink} to dismiss this alert."
+            values={{
+              schemaLink: (
+                <EuiLinkTo to={generateEnginePath(ENGINE_SCHEMA_PATH)}>
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.schemaFieldsLinkLabel',
+                    {
+                      defaultMessage: 'schema fields',
+                    }
+                  )}
+                </EuiLinkTo>
+              ),
+            }}
+          />
+        </EuiCallOut>
+      )}
+      {schemaFieldsWithConflictsCount > 0 && (
+        <EuiCallOut
+          color="warning"
+          iconType="alert"
+          title={i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.schemaConflictsBannerLabel',
+            {
+              defaultMessage: 'Disabled fields',
+            }
+          )}
+          data-test-subj="SchemaConflictsCallout"
+        >
+          <FormattedMessage
+            id="xpack.enterpriseSearch.appSearch.engine.relevanceTuning.schemaConflictsErrorMessage"
+            defaultMessage="{schemaFieldsWithConflictsCount, number} inactive {schemaFieldsWithConflictsCount, plural, one {field} other {fields}} due to field-type conflicts. {link}"
+            values={{
+              schemaFieldsWithConflictsCount,
+              link: (
+                <EuiLink href={`${DOCS_PREFIX}/meta-engines-guide.html`} target="_blank">
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.whatsThisLinkLabel',
+                    {
+                      defaultMessage: "What's this?",
+                    }
+                  )}
+                </EuiLink>
+              ),
+            }}
+          />
+        </EuiCallOut>
+      )}
```

Comment:
[Not a blocker/change request] I hadn't actually realized until QAing that there's 3 separate callouts that can appear on this page haha. What are your thoughts on potentially pulling them out to a different component for readability, e.g. `<RelevanceTuningCallouts />`? No worries if that feels overkill to you

Reply:
Actually after reviewing the unit tests maybe it's fine / not a big deal since now the callouts is the main / only thing the view has now 🤷 Maybe just an extra newline between the final/last callout and the start of the main relevance tuning form is all that's needed to help separate the sections

Reply:
https://github.com/elastic/kibana/pull/92644#discussion_r583790873 ?

Reply:
How's this? 351fd33b5a1475e39e4f1544bbc5b2fe632776a5

Reply:
Looks great!